coral/src/app/layout/header/ProfileDropdown.module.css

```diff
@@ -0,0 +1,4 @@
+.profileDropdownDeaderIcon {
+  position: relative;
+  right: -10px;
```

Comment:
It's mirroring the exact style that the console dropdown has: 

<img width="343" alt="Screenshot 2023-09-13 at 20 07 33" src="https://github.com/Aiven-Open/klaw/assets/943800/f98bd511-899b-4535-b1db-3bcef59e4038">

But it's very implicit knowledge and I'll add a comment, otherwise I myself will probably look at this at some point and be like "wait, why is that?" -> https://github.com/Aiven-Open/klaw/pull/1763/commits/80267af201ec51f7d349218212bcd1d6bacaf1e7

Reply:
I see. I don't love it in the console either hahaha! But it is what it is.